tests/integration/test_interaction.py

```diff
@@ -424,6 +472,12 @@ def test_item_no_feedback_on_bad_move(self, action_key):
         )
 
     @data(None, Keys.RETURN, Keys.SPACE, Keys.CONTROL+'m', Keys.COMMAND+'m')
+    def test_move_items_between_zones(self, action_key):
+        self.parameterized_move_items_between_zones(
```

Comment:
We were only testing that indirectly (by asserting dropped item has no `tabindex`, has a `fade` class, etc).
I added some tests to explicitly test that items cannot be moved between zones or back to the bank in standard mode.